### ISSUE TITLE
Verify newpipe.net domain on Github repo

### ISSUE THREAD

**Issue by bell-kevin** [TeamNewPipe/NewPipe#9515]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Is this the real NewPipe github repo? If yes, have we considered getting officially verified on github?

https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-organization-settings/verifying-or-approving-a-domain-for-your-organization

**Comment by opusforlife2** [comment]
Thanks for this proposal!

It's the other way around. For someone who trusts this is the real Newpipe repo, they can then further trust, based on the verified/approved tag, that the Newpipe website we link to is genuine and not fake.

Whether or not this is the official Newpipe GH repo is still something users will have to determine through their own research/common sense, based on facts like F-Droid linking here.

**Comment by bell-kevin** [comment]
![HwlROGP](https://user-images.githubusercontent.com/8269880/205995526-3309d20e-1eb3-4ce9-b3a5-f47b0821724b.png)

I'm thinking about the green verified image.

**Comment by opusforlife2** [comment]
Yes, that is basically GitHub telling you "Hey, this domain you're about to visit? It's the real deal."

**Comment by bell-kevin** [comment]
![verified](https://user-images.githubusercontent.com/8269880/206561684-f1067463-288a-4763-905b-6f5c41eb09f7.PNG)
